### ISSUE TITLE
Feature request: Released versions tags on Dockerhub

### ISSUE THREAD

**Issue by a-nldisr** [thelastpickle/cassandra-medusa#422]
**Feature request:** 
- Docker image tags currently released do not follow the versioning as on the release page: `v0.11.2` cannot be found in the dockerhub URI. When I search for s3 compatible images i either find sha strings or the master version with s3 associated to it. The latest version `v0.11.2` is nowhere to be found in the tags. This makes it a bit hard to follow the versions.

**Comment by adejanovski** [comment]
Hi @a-nldisr, here's the 0.11.2 tag: https://hub.docker.com/layers/medusa/k8ssandra/medusa/0.11.2/images/sha256-d63c8943f0bbc07108d1047f54394f378d3f29e868cb4e256a93d9f5ac2920fd?context=explore

We're not using a `v` in front of the version number in the tags, which indeed may go against what's usually done in versioning.

Is that ok with you or are you expecting tags as `v?.?.?` ?
